setup.json

```diff
@@ -16,7 +16,8 @@
     "description": "AiiDA package with utilities and interfaces for common workflows.",
     "python_requires": ">=3.5",
     "install_requires": [
-        "aiida-core~=1.2"
+        "aiida-core~=1.2",
+	"aiida-siesta"
```

Comment:
wrong indentation, anyway you need to merge/rebase to resolve the conflict

Reply:
This is a funny one because with vim editor in my local I see it perfectly indented. It's not the first time I have this problem, I need to investigate.

Reply:
your vim is probably inserting tab characters and then is not configured to display a tab-character as 4-characters